packages/compass-user-model/lib/model.js

```diff
@@ -37,8 +37,8 @@ var User = Model.extend(storageMixin, {
   }
 });
 
-User.getOrCreate = function(id, done) {
-  var user = new User({
+User.getOrCreate = (id) => new Promise((resolve, reject) => {
```

Comment:
Also big :clap: for these cleanups! Definitely feel like these are worth doing when we touch older code :)